src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -23,6 +24,18 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
     {
         private const int MaxResults = 3;
 
+        private readonly IPackageInstallerService _packageInstallerService;
```

Comment:
I don't see how these are ever set except in tests?  The derived classes have constructors that set them with comments about for testing, and then default constructors that will leave them null?